src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -388,48 +328,39 @@ private void DestroyAccounts(TransactionSubstate substate)
         {
             foreach (var toBeDestroyed in substate.DestroyList)
             {
-                if (_logger.IsEnabled(LogEventLevel.Verbose))
-                {
-                    _logger.Verbose($"Destroying account {toBeDestroyed}");
-                }
-                
+                if (_logger.IsEnabled(LogEventLevel.Verbose)) _logger.Verbose($"Destroying account {toBeDestroyed}");
+
                 _stateProvider.DeleteAccount(toBeDestroyed);
             }
         }
-        
-        private void DeployCode(ExecutionEnvironment env, TransactionSubstate substate, ref ulong unspentGas, IReleaseSpec spec)
+
+        private void DeployCode(ExecutionEnvironment env,
+            TransactionSubstate substate,
+            ref ulong unspentGas,
+            IReleaseSpec spec)
         {
             var codeDepositGasCost = (ulong) CodeDepositHandler.CalculateCost(substate.Output.Length, spec);
-            if (unspentGas < codeDepositGasCost && spec.IsEip2Enabled)
-            {
-                throw new OutOfGasException();
-            }
+            if (unspentGas < codeDepositGasCost && spec.IsEip2Enabled) throw new OutOfGasException();
+
+            if (unspentGas < codeDepositGasCost) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4614986)